### ISSUE TITLE
library`s interface

### ISSUE THREAD

**Issue by darky7669** [minischetti/metro-for-steam#20]
one little issue at library`s tab is missing the skin ui,but the buttons are showing and changing on other tabs like store and others works fine
![image](https://user-images.githubusercontent.com/85921549/212862365-ddd0a813-a93c-4807-b36d-2b4461d9cc0f.png)

**Comment by TanimsizK** [comment]
yes i have the same problem right now it happened after the update

**Comment by JoanStinson** [comment]
same issue here. anyone know how to fix it?

**Comment by JoanStinson** [comment]
it happens in the downloads section too
![image](https://user-images.githubusercontent.com/24247839/216168314-d8bbd5e1-0328-4696-bec5-aa809f7fab2a.png)

**Comment by JoanStinson** [comment]
I managed to fix it! 

To fix it go to _Program Files (x86) > Steam > skins > metro-for-steam-4.4 > resource > layout_ and open _uinavigatorpanel.layout_ with notepad and copy paste the contents from here: https://raw.githubusercontent.com/minischetti/metro-for-steam/a2e30afb37b29d90902889b148986bf7f7857697/resource/layout/uinavigatorpanel.layout 

It's thanks to RedSigma's commit in another branch.

**Comment by TheoGimenez7** [comment]
Thanks @JoanStinson ! It worked for me <3
